### PR TITLE
Update experience-insights-dashboard.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about requirement fo 20K plus seats

Reference # https://techcommunity.microsoft.com/t5/microsoft-365-blog/introducing-preview-of-experience-insights-for-microsoft-365/ba-p/3574383

Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9373

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/misc/experience-insights-dashboard.md [hunk 1]
```diff
@@ -13,7 +13,8 @@ description: "Get a periodic report of how people in your organization are using
...
+The Experience insights (preview) dashboard shows you data across usage and sentiment to give you a fuller view of your organization's experience with Microsoft 365.
+Experience insights is optimized for organizations with higher volumes of data so is only available for organizations with 20,000 plus seats. Microsoft are working on bringing the experience to smaller organizations in the future. This information and data on the dashboard will help you better understand and improve your users experience with Microsoft 365. The dashboard shows you data across usage and user sentiment, and helps give you a fuller picture of your user's overall experience. You can drill down into specific information such as feature usage for certain apps, exact feedback and Net Promoter Score (NPS) comments, and top help articles viewed by users in your organization. This info can help you identify opportunities to improve user’s Microsoft 365 products and app experiences in your organization.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/misc/experience-insights-dashboard.md]:
```suggestion
Experience insights is optimized for organizations with higher volumes of data so is only available for organizations with 20,000 plus seats. We are working on bringing the experience to smaller organizations in the future. This information and data on the dashboard will help you better understand and improve your users' experience with Microsoft 365. The dashboard shows you data across usage and user sentiment and helps give you a fuller picture of your users' overall experience. You can drill down into specific information such as feature usage for certain apps, exact feedback and Net Promoter Score (NPS) comments, and top help articles viewed by users in your organization. This info can help you identify opportunities to improve users’ Microsoft 365 products and app experiences in your organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - a few minor additional changes, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**